src/WebJobs.Script/Config/WorkerConcurrencyOptions.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class WorkerConcurrencyOptions : Hosting.IOptionsFormatter
```

Comment:
nit: can you mark this internal? Also rather than newing up instances in services, you should still be able to add this to DI via services.ConfigureOptions - it just doesn't get bound to host.json. Going this route, I'd remove the IOptionsFormatter implementation so the options aren't logged (since they're static and not meant for user config).

Reply:
Can not make it internal as it's needed on a test as input.
I think we just need services.AddSinglton() as we call default constructor. services.ConfigureOptions requires Configu call - OptionsSetup class.

Reply:
Tests have internal access so that wouldn't be an issue, however the real reason you can't make it internal is because it's defined in WebJobs.Script but needed in WebJobs.Script.Grpc, so must be public.